### PR TITLE
Week 5

### PR DESCRIPTION
**Pull request by payasbhrigu**

### CODE REVIEW

**Comment by payasbhrigu** [comment]
Assignment 5.4 (Unit Testing - Jest) Test Results:
image

**Comment by payasbhrigu** [comment]
@anilrao316

File: Week-5/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "week5",
+  "version": "1.0.0",
+  "description": "For Week5 Jest Assignment",
```
Comment (anilrao316) [inline Week-5/package.json:4]: remove this file

File: Week-5/Assignment4.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const mathOperations = require('./Assignment4')
+
+describe("Math tests",() => {
+    test('Addition of 4 and 6 should return 10', () => { 
+        expect(mathOperations.sum(4,6)).toBe(10);
+    });
```
Comment (anilrao316) [inline Week-5/Assignment4.test.js:6]: [For your learning] Add some negative test cases as well.. for ex : if you pass string to add method as in sum('1','2') result will be 12 right.. so add a functionality in the code such that data type is only numbers.. if the parameters are string return "number are not integers".. add this test case as well..

**Comment by anilrao316** [comment]
rest looks good

**Pull request merged**